packages/saber/example/assets/go_bath.css

```diff
@@ -0,0 +1,3 @@
+.chat {
+  response: nil;
+}
```

Comment:
should remove the `example/` and update the docs instead.

Reply:
> should remove the `example/` and update the docs instead.

docs should be updated, maybe existed example could make post developers knows how to keep existed features.